src/components/Checkbox.tsx

```diff
@@ -79,7 +79,6 @@ export const CheckboxListItem: FunctionComponent<CheckboxListItemProps> = ({
   return (
     <>
       <div className="govuk-checkboxes__item">
-        <input id={`${id}Hidden`} type="hidden" name={name} value="false" />
```

Comment:
This hidden input with the same name as the visible input was resulting in form data looking like `[ "true", "false" ]` when a checkbox was checked and just `"false"` when it wasn't.  I couldn't work out why the hidden input was there?  

Tests pass without it post-refactor.

Reply:
The hidden element was added in [this PR ](https://github.com/mcagov/beacons-webapp/pull/253) to workaround the domain object data structure.  With the mapper on this page now, the bug isn't experienced, so this line of code can be removed :-)